thirdeye/thirdeye-frontend/app/pods/rootcause/template.hbs

```diff
@@ -46,6 +46,50 @@
     }}
   </div>
 
+  <div class="row">
+    <div class="card-container card-container--box-shadow">
+      <div class="card-container__body">
+        <div class="row">
+          <div class="col-xs-3">
+            {{rootcause-legend
+              entities=entities
+              selectedUrns=selectedUrns
+              invisibleUrns=invisibleUrns
+              onVisibility=(action "onVisibility")
+              onSelection=(action "onSelection")
+            }}
+          </div>
+          <div class="col-xs-9">
+            <div class="container container-fluid">
```

Comment:
fyi, you will get some merge conflicts from my legend changes. Let me know if you need help resolving the conflics. (my changes added a wrapper and borders around the legend and tooltip.), some of the styling changes you made here should not be needed anymore

Reply:
fixed